package.json

```diff
@@ -15,6 +15,8 @@
     "build:packages": "turbo run build --filter=!docs --filter=!template-*",
     "ci:test": "./scripts/tests-ci.sh",
     "pretest": "turbo run pretest",
+    "depcheck": "knip --dependencies --tags=-knipignore",
```

Comment:
The fact that `knip` does more than a `depcheck` seems like an argument in _favor_ of `knip:check`, no?

The pattern `<toolname>:check` also goes well with `prettier:check`, `forc:check`, etc.

Reply:
Oh, is it because it's _only_ checking deps atm?

Nevertheless, could this change if [this](https://github.com/FuelLabs/fuels-ts/issues/2027) goes through?

Reply:
> s it because it's _only_ checking deps atm?
> 
> Nevertheless, could this change if [this](https://github.com/FuelLabs/fuels-ts/issues/2027) goes through?

That's correct, I was planning to change it in the PR for https://github.com/FuelLabs/fuels-ts/issues/2027

Reply:
> The pattern <toolname>:check also goes well with prettier:check, forc:check, etc.

+1 for the pattern, but I can appreciate the reasoning.